.github/workflows/publishpackage.yml

```diff
@@ -11,8 +11,8 @@ jobs:
   build:
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: [3.6, 3.7, 3.8]
+        os: [ubuntu-18.04, ubuntu-latest, windows-latest, macos-latest]
```

Comment:
What is the reason for choosing the minimum Linux version? This is because glibc "controls" everything. There is some backward compatibility. That is, we can take Ubuntu 18.04 (this is LTS) and get the minimum version of glibc (2.27 https://launchpad.net/ubuntu/bionic/+source/glibc ). The so libraries will be able to run on newer versions of glibc.

By the way, ubuntu-latest is Ubuntu 20.04 LTS. https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources.

I suggest using the minimal version for wheel too. Or all versions.

Reply:
Oh, okay, that sounds pretty reasonable. If we can rely on ubuntu's backward comp and we can use all necessary python versions with it - I have no objections to use 18.04 as base version for linux build. 👍 🙂